include/aws/http/private/h2_connection.h

```diff
@@ -143,6 +156,9 @@ enum aws_h2_data_encode_status {
 
 /* When window size is too small to fit the possible padding into it, we stop sending data and wait for WINDOW_UPDATE */
 #define AWS_H2_MIN_WINDOW_SIZE (256)
+/* We will ignore frames for closed stream within this time slot, after that we will treat them as connection error:
+ * nano secs, 100000000 nano secs -> 0.1 sec */
+#define AWS_H2_IGNORE_TIME (100000000)
```

Comment:
I don't really know how long it should be, and the spec seems let us make our own choice.

Reply:
We should  see if we can get some inspiration from other H2 implementations, check node's or chrome's maybe?

Reply:
So, I looked into Netty and Nghttp2 to see how they deal with the close stream. And they both not exactly follow the spec to define a time slot for ignoring the frame...🤦‍ And Chrome is just too complicated...

Reply:
Used LRU cache instead of a fixed time.

Reply:
Oh, one possible case? Someone open a lot of streams and keep sending RST_STREAM on them, the LRU cache will keep those closed streams as MRU, and the other recently closed stream will be kept out of the cache, and we will result in treating all the others as connection error.🤦‍And we will keep sending RST_STREAM back...

Reply:
Finially, FIFO cache instead of LRU cache!